client-app/src/desktop/tabs/examples/ExamplesTab.js

```diff
@@ -1,138 +1,230 @@
-                </p>
-            ]
-        },
-        {
-            title: 'Contact',
-            icon: Icon.users(),
-            path: '/contact',
-            text: [
-                <p>
-                    Meet the Extremely Heavy team!
-                </p>,
-                <p>
-                    This example shows an employee directory application. Includes multiple views of a store, including
-                    combined filter and search functionality, editable profiles and remote persistence of settings via
-                    soft config.
-                </p>
-            ]
-        },
-        {
-            title: 'Todo',
-            icon: Icon.clipboard(),
-            path: '/todo',
-            text: [
-                <p>
-                    The classic reference app, Hoist style.
-                </p>,
-                <p>
-                    Includes examples of a grid with RecordActions and RecordActionBar, a form with validation,
-                    modal dialogs, and the preference system.
-                </p>
-            ]
-        },
-        {
-            title: 'News',
-            icon: Icon.news(),
-            path: '/news',
-            text: [
-                <p>
-                    This example demonstrates Hoist support for loading and caching data on the server from
-                    a {link('Remote API', 'https://newsapi.org/')}. Refresh rate, news sources, and API key can
-                    be modified in the Admin Config tab.
-                </p>,
-                <p>
-                    On the client side, we use a {link(code('DataView'), '../app/grids/dataview')} grid
-                    to support custom filtering logic and rich component rendering.
-                </p>
-            ]
-        },
-        {
-            title: 'FDA Recalls',
-            icon: Icon.health(),
-            path: '/recalls',
-            text: [
-                <p>
-                    This applet uses the openFDA drug enforcement reports API, which provides information on drug recall
-                    events since 2004. Provides examples of filtering and searching data from an external API.
-                </p>,
-                <p>
-                    For more information, see {link('here', 'https://open.fda.gov/apis/drug/enforcement/')}.
-                </p>
-            ]
-        },
+
+const sideBar = hoistCmp.factory(({model}) => {
+    return toolbar({
+        className: 'app-toolbar',
+        items: [...model.examples.map(e => button({
+            icon: e.icon,
+            onClick: () => model.activeApp === e.title ? window.open(e.path) : model.setActiveApp(e.title)
+        })),
+        button({
+            icon: Icon.chevronRight(),
+            onClick: () => model.leftPanelModel.collapsed = false
+        })],
+        vertical: true
+    });
+});
+
+class LocalModel extends HoistModel {
```

Comment:
And this model should go in its own file --   Its complex and important enough -- not an implementation detail for the component.